core/modules/qdisp/Executive.h

```diff
@@ -136,12 +149,20 @@ class Executive {
     std::shared_ptr<MessageStore> _messageStore; ///< MessageStore for logging
     XrdSsiService* _service; ///< RPC interface
     RequesterMap _requesters; ///< RequesterMap for results from submitted tasks
-    StatusMap _statuses; ///< Statuses of submitted tasks
+    JobStatusPtrMap _statuses; ///< Statuses of submitted tasks
+
+    /** Execution errors */
```

Comment:
let's do comments consistently. Looks like a bit higher we are using "/// < text"